web/tests/test_message_students.py

```
@@ -0,0 +1,166 @@
+        # Enroll students in the course
+        Enrollment.objects.create(student=self.student1, course=self.course, status="approved")
+        Enrollment.objects.create(student=self.student2, course=self.course, status="approved")
+
+        # Set up the client and login as teacher
+        self.client = Client()
+        self.client.login(username="teacher", password="password123")
+
+        # URL for message_enrolled_students view
+        self.url = reverse("message_students", kwargs={"slug": self.course.slug})
+
+    def test_get_message_form(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_get_message_form`

Add return type annotation: `None`

(ANN201)